osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -121,10 +121,13 @@ private double computeAimValue()
                                 : 0.0);
             }
 
-            // Scale the aim value with accuracy _slightly_
-            aimValue *= 0.5 + accuracy / 2.0;
+            // Scale the aim value with accuracy _a lot_
+            if (accuracy > .8)
+                aimValue *= 0.5 + Math.Pow(Math.Sin(2.5 * (accuracy - .8) * Math.PI), 2) / 2;
```

Comment:
I'm not sure I can meaningfully comment on the general validity of this curve change, but I find the actual shape/plot a little... arbitrary? Is there a particular reason why the piece at and above 0.8 is a sin<sup>2</sup> instead of, say, another linear, or quadratic piece?

If it's to keep smoothness, then that's not really happening, as the function is not smooth (first derivative is not continuous as far as I can tell - it goes from 5/8ths on the left side of 0.8 to 0 on the right side).

Not entirely sure why small changes in accuracy in the middle part of that sine curve part should result in larger changes than at its ends.